docs/source/UserGuide/AdvancedConfiguring/using-for-on-premise.md

```diff
@@ -170,8 +170,8 @@ You also need to provide ssh_proxy_command if the head node needs to access the
 auth:
     ssh_user: [sudo user]
     # Specify the private key file for login to the nodes
-    # use `ssh-keygen -t rsa -b 4096` to generate a new ssh key pair on head node.
-    # Then add head SSH public key to each workers. For example: ssh-copy-id -i ~/.ssh/id_rsa [sudo user]@[worker-ip]
+    # use `ssh-keygen -t rsa -b 4096` to generate a new ssh key pair on working node.
+    # Then add working node SSH public key to each nodes. For example: ssh-copy-id -i ~/.ssh/id_rsa [sudo user]@[node-ip]
     ssh_private_key: ~/.ssh/id_rsa
```

Comment:
The comments are easy to be ignored for users reading doc.
In stead of keep these in comments, shall we move these to "### Create a new sudo user and setup login with private key" section for giving a user a practical guide of how to setup the login with a private key?

Reply:
@jerrychenhf Yes. This step is necessary for using on-premise mode. I will add this requirement as following step:
```
1.      All the machines needs to have a user with sudo privilege and private key login. A non-root user is suggegested, for example ‘cloudtik’.
2.      Setup host resolution for all the nodes
3.      Prepare the private key file for the clusters
4.      Prepare the local disks```